vms/vms/static/vms/js/hide_resume.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+    $("input[type='file']").change(function() {
+    var FileLength = $("#id_vol-resume_file")[0].files.length;
```

Comment:
variable names in JS follows `camelCase`

Reply:
Also, I wouldnt name it hide_resume as its actually hide_resume_textbox - minor though.

Reply:
@tonythomas01 Done with the changes you suggested. Can you please review.